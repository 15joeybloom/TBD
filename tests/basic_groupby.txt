select count(GradYear) from student group by GradYear
