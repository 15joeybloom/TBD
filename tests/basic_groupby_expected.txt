
SQL> 
select count(GradYear) from student group by GradYear
count(gradyear)
------
     4
     2
     1
     2

SQL> 
exit
